cli/environment-builder.js

```diff
@@ -0,0 +1,299 @@
+const assert = require('assert');
+const chalk = require('chalk');
+const _ = require('lodash');
+const path = require('path');
+const Environment = require('yeoman-environment');
+const { CLI_NAME, logger } = require('./utils');
+const { normalizeBlueprintName, packageNameToNamespace, loadYoRc, loadBlueprintsFromConfiguration } = require('../generators/utils');
+
+module.exports = class EnvironmentBuilder {
```

Comment:
can you add a comment saying what a JHipster environment is? i.e. standard yo env + blueprints
Maybe this class should be renamed JHipsterEnvironmentBuilder, or something

Reply:
I had problems before putting jsdoc in the class, they were duplicating.
I had to move to the constructor to fix that.